src/pages/home/ReportScreen.js

```diff
@@ -216,15 +216,19 @@ class ReportScreen extends React.Component {
 
         // There are no reportActions at all to display and we are still in the process of loading the next set of actions.
         const isLoadingInitialReportActions = _.isEmpty(this.props.reportActions) && this.props.report.isLoadingReportActions;
+
+        const freeze = this.props.isSmallScreenWidth && this.props.isDrawerOpen;
+        const animatePlaceholder = !freeze;
```

Comment:
Can you give some context for this?

Also can you please include some explanation in the PR body?